docs/meetings/2023/TechArea20231128.md

```diff
@@ -49,6 +50,7 @@ Miscellaneous:
 Remaining OSG 23 items <https://opensciencegrid.atlassian.net/browse/SOFTWARE-5503>.
 -   Figure out if we need VOMS for EL9/an update to EL8
 -   AI (Tim): Update tarball scripts to use `23-main` (SOFTWARE-5713)
+    - Done in previous release. Changing from 'latest' releases to concious choice to change version, possibly in Febuary?
```

Comment:
```suggestion
```

Dropping this since the work was actually done. Dropping the `latest` symlink is orthogonal and more of a policy decision